### DISCUSSION TITLE
Junior TV

### DISCUSSION THREAD

**Discussion by Agumonz98** [iptv-org/community#739] [Q&A]
**Channel Name:** Junior TV
**Channel ID:** JuniorTV.al
**Country:** Albania
**Language:** Albanian
**Website:** http://www.digitalb.al/
**Stream URL (optional):** (Don't have one)
**Notes (optional):** Again I'm just look for a good working stream for this channel, doesn't matter if its a live-stream or m3u8. (Also all the Albanian/Macedonian streams from #4986 don't seem to work anymore)

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19735
